js/components/App.js

```diff
@@ -1,13 +1,27 @@
 import 'babel/polyfill';
 
+// Mutations
+import ToggleWidgetEnableMutation from '../mutations/ToggleWidgetEnableMutation';
+
 class App extends React.Component {
+  _handleToggleEnabled(widget) {
+    Relay.Store.update(new ToggleWidgetEnableMutation({
+      widget: widget,
+    }));
```

Comment:
Can this just be `Relay.Store.update(new ToggleWidgetEnableMutation({widget}));`?